index.html

```diff
@@ -174,5 +175,37 @@ <h1>I'm Shimels, Glad to see you!</h1>
         </li>
       </ul>
     </section>
+    <section id="contact-form">
+      <h1>Contact me</h1>
+      <p>
+        If you have an application you are interested in developing, a feature
+        that you need built or a project that needs coding. I’d love to help
+        with it!
+      </p>
+      <form action="https://formspree.io/f/xyyqzzjg" method="post">
+        <input
+          type="text"
+          id="name"
+          name="user_name"
+          placeholder="Write your name here"
+          maxlength="30"
+          required
+        />
+        <input
+          type="email"
+          id="email"
+          name="user_email"
+          placeholder="Write your email here"
+          required
+        />
+        <textarea
+          name="user_message"
+          placeholder="Write your message here"
+          maxlength="500"
+          required
+        ></textarea>
+        <button type="submit" class="content-submit-btn">Get in touch</button>
```

Comment:
- [ ] 👍 Your button looks great with proper font styles and color! However, I noticed that the padding is currently set to 12px on all sides, which does not align with the design specifications.  To achieve a closer match with the design, please consider updating your code to adjust the padding as per the design requirements. Make sure to use the appropriate padding values on each side of the button to achieve the desired appearance.

![image](https://github.com/shimels1/portfolio_microverse/assets/55840999/416e446e-5b94-41b0-a2d5-1f32fd5b5bbe)

By making this adjustment, your button will be visually consistent with the design, enhancing the overall aesthetics of your project. Keep up the good work, and I'm confident your attention to detail will lead to a polished final result! 🎨🚀